packages/x-scheduler/src/material/timeline/Timeline.types.ts

```diff
@@ -1,6 +1,10 @@
 import { TimelineParameters } from '../../primitives/use-timeline';
+import { ViewSwitcherTranslations } from '../models/translations';
 
-export type TimelineView = 'hours' | 'days' | 'weeks' | 'months' | 'years';
+export type TimelineView = Extract<
```

Comment:
I think you can just hardcode the list here, extracting it from `ViewSwitchingTranslations` doesn't bring a lot of benefits